engine/rpcserver.go

```diff
@@ -1874,14 +1905,29 @@ func (s *RPCServer) GetAuditEvent(_ context.Context, r *gctrpc.GetAuditEventRequ
 
 // GetHistoricCandles returns historical candles for a given exchange
 func (s *RPCServer) GetHistoricCandles(_ context.Context, r *gctrpc.GetHistoricCandlesRequest) (*gctrpc.GetHistoricCandlesResponse, error) {
-	if r.Exchange == "" {
-		return nil, errExchangeNameUnset
+	if r.Start >= r.End {
```

Comment:
`r.Start` and `r.End` are `string` type.  I don't think this should be the basis of comparing times.